ecs-logs-collector.sh

```diff
@@ -271,6 +271,10 @@ get_sysinfo() {
       pkgtype="deb"
       if grep --quiet "Ubuntu 14.04" /etc/${found_file}; then
         os_name="ubuntu14"
+      elif grep --quiet "Ubuntu 16.04" /etc/${found_file}; then
+        os_name="ubuntu16"
```

Comment:
why are these the same?

Reply:
Because they are Systemd based OS. It's needed to know where to get the logs from later on. We could set "ubuntu18" but we would need to add each new version here then while they are the same:

`amazon2|redhat|debian|ubuntu16)`